pkg/knative/utils.go

```diff
@@ -0,0 +1,66 @@
+package knative
+
+import (
+	"encoding/json"
+	"net/http"
+
+	apiextensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/coreos/pkg/capnslog"
+	"github.com/openshift/console/pkg/serverutils"
+)
+
+var (
+	plog = capnslog.NewPackageLogger("github.com/openshift/console", "knative")
+)
+
+// EventSourceVersion describes a version for CRD.
+type EventSourceVersion struct {
+	Name    string `json:"name" protobuf:"bytes,1,opt,name=name"`
+	Served  bool   `json:"served" protobuf:"varint,2,opt,name=served"`
+	Storage bool   `json:"storage" protobuf:"varint,3,opt,name=storage"`
+}
+
+// EventSourceSpec describes how a user wants their resource to appear
```

Comment:
Nit, you could put the `types` in a `types.go` and the utils in a separate file, but not a big deal unless you keep adding to this.

Reply:
Agree. Hope its fine if I consider it in next change set!